src/client/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -444,6 +444,11 @@ internal override void Validate()
             {
                 throw new InvalidOperationException(MsalErrorMessage.InvalidRedirectUriReceived(Config.RedirectUri));
             }
+
+            if (Config.IsBrokerEnabled && Config.IsWebviewSsoPolicyEnabled)
+            {
+                throw new MsalClientException(MsalErrorMessage.BrokerAndSsoPolicyConfigurationError);
+            }

```

Comment:
Should this be removed? If someone specifies WithBroker and WithSsoPolicy, let the broker handle the SSO stuff.

Reply:
yes, we had a discussion about this offline with WAM. This indeed is not needed because we might need to provide fallback logic in case broker is not installed even when apps use broker